File: hw/viteri/u2/HW18ZooInPython/Zoo/model/Eagle.py
from datetime import date
from model.Bird import Bird
from model.Cage import Cage

class Eagle(Bird):
    def __init__(self, id, description, born_on_date, gender, cage, number_of_bones, vertebrate):
        super().__init__(id, description, born_on_date, gender, cage, number_of_bones, vertebrate)

    def feed(self, food_amount):
        print(f"Feeding the eagle with {food_amount} units of food")

File: hw/segarra/u2/HW18ZooInPythonAndJavaScript/ec.edu.zooinjavascript.model/Eagle.js
import Bird from "./Bird.js"

export default class Eagle extends Bird{
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate){
        super(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate)
    }

    feed(){
        console.log('Feeding the eagle with mice')
    }

}